client/src/components/exercise/lab1/components/Countdown.js

```diff
@@ -12,7 +12,7 @@ class Countdown extends Component {
 			<Fragment>
 				<CongratulationMessage message={message} />
 
-				<div className="game__countdown">{time}</div>
+				<div className="Exercise__countdown">{time}</div>
```

Comment:
Idk if this makes a difference, but could "Exercise__countdown" be lowercase? Just to follow the convention before of "game__countdown"